tsfc/fem.py

```diff
@@ -75,23 +71,26 @@ def tabulate(ufl_element, order, points):
         yield c, D, table
 
 
-def make_tabulator(points):
-    """Creates a tabulator for an array of points."""
-    return lambda elem, order: tabulate(elem, order, points)
+def make_tabulator(points, epsilon):
+    """Creates a tabulator for an array of points with rounding
+    parameter epsilon."""
+    return lambda elem, order: tabulate(elem, order, points, epsilon)
```

Comment:
This parametrization/configuration could have a much shorter diff if Python had support for dynamic scoping.